README.md

```diff
@@ -25,8 +25,9 @@ jobs:
       - name: match-tag-to-package-version
         id: match-tag-to-package-version
         uses: geritol/match-tag-to-package-version@0.1.0
-        env:
+        with:
```

Comment:
Not sure what do we gain with also allowing to set the tag prefix as an input

Reply:
The gain is flexibility.  I have a workflow where this is the only action that (currently) needs an `env` key.  All the other actions in the workflow use the `with` key to pass in data.  At first, my workflow failed at this step because I used `with` not `env` (guess I didn't read the docs)!  I don't have enough experience with GitHub Actions to really know if there is a standard, but I figured why not allow either.